core/static/js/dit.class.modal.js

```diff
@@ -166,6 +163,18 @@
     }
   }
 
+  Modal.bindKeyboardFocusEvents = function() {
```

Comment:
Perhaps it would be better to have a separate self.$lastFocusElement.on("keydown") specified/applied in this function to keep that trapping functionality together. It would also make things more readable and easier to understand the interaction between first and last.